### DISCUSSION TITLE
Repository

### DISCUSSION THREAD

**Discussion by Tamunoib** [community/community#102934] [Repositories] [answered]
### Select Topic Area

Question

### Body

I don't have access to a repository I created six months ago, anytime I try to Git Push to my GitHub and into that repository, it seems the repository is not recognized while it is in my GitHub account. How do I resolve this issue?

**Comment by DaviZCodes** [comment, marked as answer]
There could be many reasons for this issue.

First, check if you are in the correct branch. You could be trying to push to another branch instead of the main one. 

Also, there could be several copies of your cloned repository on the computer. Check if you are in the right one. If you have too many, delete all the folders in your computer, and then git clone again. 

How are you trying to access the repository? Are you using the terminal or GitHub Desktop? If you keep having issues with the terminal, I recommend downloading GitHub Desktop and opening the repository through the app GitHub Desktop - which saves time and effort.

**Reply by Tamunoib** [reply]
Thanks very much, noted. How do I recover a passphrase key that I have forgotten

**Reply by DaviZCodes** [reply]
Try: https://docs.github.com/en/authentication/troubleshooting-ssh/recovering-your-ssh-key-passphrase

I think you should just create a new key.
